functions/ldap.js

```diff
@@ -182,8 +212,28 @@ exports.plugin = {
               client.search(base, options, (error, result) => {
                 let rows = [];
 
-                function normalizedRows() {
-                  return !raw ? simplify(rows) : rows;
+                function normalizeRows() {
+                  try {
+                    rows.forEach((row) => {
```

Comment:
```suggestion
                    for (const row of rows) {
```

Prefer `for...of` loops.

Reply:
implemented to all instances